scripts/export_check/compatibility_check.py

```diff
@@ -12,24 +12,27 @@
     from pathlib import Path
     proj_path = os.path.abspath(Path(__file__).parents[2])
     sys.path.append(proj_path)
+    sys.path.append('vortex/development_package')
+    sys.path.append('vortex/runtime_package')
```

Comment:
I have tested this, and this is a false path export. The import `os`, `sys`, and `pathlib` is redundant at this point. 
Also, this script is better renamed to `onnx_check.py` to be honest. cc @alifahrri

Reply:
need suggestion for this please

Reply:
already fixed